Engine/ac/sys_events.cpp

```diff
@@ -794,10 +860,14 @@ static void on_sdl_touch_up(const SDL_TouchFingerEvent &event)
     }
     default: break; // do nothing
     }
+    touch.fingers.pop(event.fingerId);
 }
 
 static void on_sdl_touch_motion(const SDL_TouchFingerEvent &event)
 {
+    int finger_index = touch.fingers.get_index(event.fingerId);
```

Comment:
Do you have thoughts on `tfinger_to_mouse_but` using finger_index instead of fingerId? If it used finger_index then I don't need to ask for it again inside of it.

Reply:
Oh, sure.

Reply:
OK, I added this in the commit below